lxd/instance/drivers/driver_lxc_cmd.go

```diff
@@ -36,20 +35,11 @@ func (c *lxcCmd) Signal(sig unix.Signal) error {
 func (c *lxcCmd) Wait() (int, error) {
 	err := c.cmd.Wait()
 	if err != nil {
-		exitErr, ok := err.(*exec.ExitError)
-		if ok {
-			status, ok := exitErr.Sys().(syscall.WaitStatus)
-			if ok {
-				return status.ExitStatus(), nil
-			}
-
-			if status.Signaled() {
-				// 128 + n == Fatal error signal "n"
-				return 128 + int(status.Signal()), nil
```

Comment:
@brauner what I'm not clear about was whether we ever ran this code path, as presumably if we called `status.Signaled()` bit on `status` when it wasn't of type `syscall.WaitStatus` we would get a panic. It seems like this was a dead code path.